source/Sashimi.Tests.Shared/Sashimi.Tests.Shared.csproj

```diff
@@ -7,7 +7,6 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <Nullable>enable</Nullable>
-    <IsPackable>false</IsPackable>
   </PropertyGroup>
```

Comment:
I think we'll want the project/license information here as well, like:
https://github.com/OctopusDeploy/Sashimi/blob/93e64d66c5b328a0e86f2dcecf68257096c2a665/source/Sashimi.Terraform/Sashimi.Terraform.csproj#L11-L12